components/html-block/html-block.js

```diff
@@ -7,6 +7,12 @@ import { loadMathJax } from '../../helpers/mathjax.js';
  */
 class HtmlBlock extends LitElement {
 
+	static get properties() {
+		return {
+			mathJaxConfig: { type: Object, attribute: 'math-jax-config' }
```

Comment:
I don't think we should surface "MathJax" specifically on the `d2l-html-block` API - the fact that we are using MathJax should be kept internal from our app components.  That aside, exposing the LaTeX aspect on `d2l-html-block`'s API allows for inconsistent UX that would be frustrating for users, and also introduces a bunch of complexity for those scenarios (ex. more than one `d2l-html-block` on the same page with different math settings).

Instead, I was thinking that we would internally check the client-side feature flag for the instance/org.  In effect, this is passed "under the covers" without exposing it on any APIs.

Reply:
Oh, I see what you mean. Sure, yeah. Something like `if (D2L.LP && {FlagCheck}) ...` should be easy enough to do here. I suppose that won't work in FRAs though without designing a pass-through for flags.

Reply:
Yeah so just to close the look on this - when/if we need to support this in a FRA, we'll need to pass the proper config through ifrau to it can be used in the FRAs context.

Reply:
Thinking about ifrau makes me sad, Dave.